src/api/todos.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodos = (data: Omit<Todo, 'id'>) => {
+  return client.post<Todo>(`/todos?userId=${data.userId}`, data);
+};
+
+export const deleteTodos = (todoId: number) => {
```

Comment:
For consistency, consider changing 'deleteTodos' to 'deleteTodo' as it deletes a single todo item.